### PR TITLE
Presets bugfix: automatically preselect current FC firmware version

### PR DESCRIPTION
**Pull request by limonspb**
It was intended that the Firmware selection dropdown automatically selects the current firmware version of the plugged in FC:
image
However, because of the bug, it did not select anything showing all the presets by default regardless of the Firmware version the user plugged in.
The reason is the multipleSelect component hack for 199+ items that calls 'refresh' in 100ms after the page load.
The solution is to preselect the firmware version after this timeout, not before. Together with the other initializations.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/tabs/presets/presets.js
Comment (chmelevskij) [inline src/tabs/presets/presets.js:348]: Good approach, but we are hooking into time base flakey event anyways 😄 Wondering if there are better events somewhere to hook in.

Comment (limonspb) [reply src/tabs/presets/presets.js:348]: Yeah unfortunately because of the bug in drop-down component we have to timout for 100ms...

**Review by haslinghuis — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Pull request merged**
